.github/dependabot.yml

```
@@ -23,7 +23,7 @@ updates:
     labels:
       - ci
       - dependency
-      - github-actions
+      - github_actions
```

```suggestion
      - github-actions
```
https://github.com/jellyfin/jellyfin-meta-plugins/blob/acc62bb34aaa87efc2560434a384b50a95d8df82/.github/plugin-repo-labels.yaml#L114